src/pages/BookDetail/BookDetail.tsx

```diff
@@ -10,30 +10,46 @@ import { fetchRecommendedBooks } from '../../api/recommendedBooksApi';
 import { Loading } from '../../components/Loading/Loading';
 import { RecommendedBooks } from './RecommendedBooks';
 import { getDescriptionValue } from '../../utils/getDescriptionValue';
+import { Button } from '../../components/Button/Button';
+import { SelectClubDialog } from '../../components/ConfirmDialog/SelectClubDialog';
+import { useAuth } from '../../context/AuthContext';
+import { isOwnerOfClub } from '../../api/clubsApi';
 import style from './BookDetail.module.scss';
 
 const iconSx = { fontSize: 16 };
 
 export const BookDetail: React.FC = () => {
   const { id: editionId } = useParams();
   const [searchParams] = useSearchParams();
+  const user = useAuth();
   const authorKey = searchParams.get('authorKey');
+  const clubId = searchParams.get('clubId');
```

Comment:
Dont separate the related lines with unrelated ones
```suggestion
  const [searchParams] = useSearchParams();
  const authorKey = searchParams.get('authorKey');
  const clubId = searchParams.get('clubId');
    const user = useAuth();
```